js/viz.js

```diff
@@ -86,8 +86,8 @@ const viz = {
     this.context.scale(this.scale, this.scale);
   },
 
-  getDimensions(id) {
-    const element = document.getElementById(id);
+  getDimensions() {
+    const element = document.body;
     const { width, height } = element.getBoundingClientRect();
 
```

Comment:
because we are calculating width, height w.r.to the body element, and the graph uses width/2, height/2 as the centre point, we have the following issue for resize:

![screen shot 2017-08-22 at 14 32 33](https://user-images.githubusercontent.com/8022693/29565629-f9d51b5e-8746-11e7-8824-9b67c594bf65.png)

The graph is not really centred now w.r.to the vis area. Will file a follow-up issue..